internal/compiler/generator/rust.rs

```diff
@@ -782,6 +782,24 @@ fn generate_sub_component(
                 }
             }
 
+            fn next_in_local_focus_chain(self: ::core::pin::Pin<&Self>, _index: usize) -> usize {
+                // Implement this!
+                0
```

Comment:
in rust speak, that would be

```suggestion
                todo!()
```

Reply:
I put "unimplemented!()", but that of course broke all the rust tests:-)